connector/src/main/java/org/geysermc/connector/utils/SkinProvider.java

```diff
@@ -53,8 +55,8 @@
     public static final Skin EMPTY_SKIN = new Skin(-1, "steve", STEVE_SKIN);
     public static final byte[] ALEX_SKIN = new ProvidedSkin("bedrock/skin/skin_alex.png").getSkin();
     public static final Skin EMPTY_SKIN_ALEX = new Skin(-1, "alex", ALEX_SKIN);
-    private static Map<UUID, Skin> cachedSkins = new ConcurrentHashMap<>();
-    private static Map<UUID, CompletableFuture<Skin>> requestedSkins = new ConcurrentHashMap<>();
+    private static Map<String, Skin> cachedSkins = new ConcurrentHashMap<>();
```

Comment:
There's a couple of imports here that can be marked as final - might as well clean that up now if possible.